<html>
<head>
<title>TestNG:  Tutorial Ninja functional Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Tutorial Ninja functional Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>8/2/4</td>
</tr><tr>
<td>Started on:</td><td>Tue Sep 19 17:48:06 EDT 2023</td>
</tr>
<tr><td>Total time:</td><td>62 seconds (62831 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.liginPagefail()'><b>liginPagefail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1101179754", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1101179754'><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>7</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.draftfail()'><b>draftfail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2054138033", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2054138033'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1067)
	at org.testng.Assert.assertEquals(Assert.java:1031)
	at org.testng.Assert.assertEquals(Assert.java:1077)
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality.r1dot1_TS003_TC001()'><b>r1dot1_TS003_TC001</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality<br>Test method: R1.1_TS003_TC001 Verify Logging out by selecting Logout option from 'My Account' drop menu</td>
<td></td>
<td>32</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS003_LogoutFunctionality@3bf7ca37</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC028()'><b>r1dot1_TS001_TC028</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality<br>Test method: R1.1_TS001_TC028 Verify the application throws proper error message when user click continue button without selecting privacy and policy check box.</td>
<td></td>
<td>9</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@7c6908d7</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality.r1dot2_TS006_TC001()'><b>r1dot2_TS006_TC001</b><br>Test class: com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality<br>Test method: R1.2_TS006_TC001 As an user I should be able to select products for comparison.</td>
<td></td>
<td>18</td>
<td>com.tutorialNinja.functionalTests.R1dot2_TS006_ProductCompareFunctionality@79079097</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC003()'><b>r1dot1_TS001_TC003</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality<br>Test method: R1.1_TS001_TC003 Verify Registering an Account by providing all the fields</td>
<td></td>
<td>14</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@7c6908d7</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality.r1dot1_TS002_TC001()'><b>r1dot1_TS002_TC001</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality<br>Test method: R1.1_TS002_TC001 Verify logging into the Application using valid credentials</td>
<td></td>
<td>20</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS002_LoginFunctionality@3c9754d8</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC001()'><b>r1dot1_TS001_TC001</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality<br>Test method: R1.1_TS001_TC001 As a user I should be able to Register an Account by providing only the Mandatory fields</td>
<td></td>
<td>22</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@7c6908d7</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality.r1dot2_TS005_TC001()'><b>r1dot2_TS005_TC001</b><br>Test class: com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality<br>Test method: R1.2_TS005_TC001 As an user I should be able to search a product which is present in database.</td>
<td></td>
<td>11</td>
<td>com.tutorialNinja.functionalTests.R1dot2_TS005_ProductSearchFunctionality@4d1c00d0</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality.r1dot1_TS001_TC004()'><b>r1dot1_TS001_TC004</b><br>Test class: com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality<br>Test method: R1.1_TS001_TC004 Verify proper error messages are displayed for the mandatory fields, when you don't provide any fields in the 'Register Account' page and submit</td>
<td></td>
<td>9</td>
<td>com.tutorialNinja.functionalTests.R1dot1_TS001_RegisterFunctionality@7c6908d7</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.liginPagefail()'><b>liginPagefail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1223811987", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1223811987'><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>9</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.draftfail()'><b>draftfail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1954213225", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1954213225'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1067)
	at org.testng.Assert.assertEquals(Assert.java:1031)
	at org.testng.Assert.assertEquals(Assert.java:1077)
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:248)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.draftfail()'><b>draftfail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [true] but found [false]
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 19 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace97592792", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace97592792'><pre>java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:1067)
	at org.testng.Assert.assertEquals(Assert.java:1031)
	at org.testng.Assert.assertEquals(Assert.java:1077)
	at com.tutorialNinja.functionalTests.Draft.draftfail(Draft.java:15)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>0</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
<tr>
<td title='com.tutorialNinja.functionalTests.Draft.liginPagefail()'><b>liginPagefail</b><br>Test class: com.tutorialNinja.functionalTests.Draft</td>
<td><div><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 18 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2064783608", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2064783608'><pre>java.lang.AssertionError: expected [mac] but found [Account Login]
	at org.testng.Assert.fail(Assert.java:111)
	at org.testng.Assert.failNotEquals(Assert.java:1578)
	at org.testng.Assert.assertEqualsImpl(Assert.java:150)
	at org.testng.Assert.assertEquals(Assert.java:132)
	at org.testng.Assert.assertEquals(Assert.java:656)
	at org.testng.Assert.assertEquals(Assert.java:666)
	at com.tutorialNinja.functionalTests.Draft.liginPagefail(Draft.java:25)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
</pre></div></td>
<td>12</td>
<td>com.tutorialNinja.functionalTests.Draft@2d1ef81a</td></tr>
</table><p>
</body>
</html>